app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -72,6 +92,63 @@ class Identification extends React.Component {
       });
   }
 
+  getLanguageOptions(languageType) {
+    const langOptions = supportedLanguages.languages.map(function(lang) {
+      let fullySupported = lang.supported.sms && lang.supported.email && lang.supported.phone;
+      let langLabel = fullySupported ? lang.name : lang.name + '*';
+      return { value: lang.name, label: langLabel, type: languageType };
+    });
+    return langOptions;
+  }
+
+  getLanguageValue(language, languageType) {
+    return this.getLanguageOptions(languageType).filter(lang => lang.value === language);
+  }
+
+  renderLanguageSupportMessage(selectedLanguage, languageType) {
+    if (selectedLanguage) {
+      let languageJson = supportedLanguages.languages.find(l => l.name === selectedLanguage);
+      let sms = languageJson.supported.sms;
+      let email = languageJson.supported.email;
+      let phone = languageJson.supported.phone;
+      let fullySupported = sms && email && phone;
+
+      if (!fullySupported) {
+        let message = languageJson.name;
+        if (!sms && !email && !phone) {
+          message += ' is not currently supported by Sara Alert.';
+          if (languageType === 'primary') message += ' Any messages sent to this monitoree will be in English.';
```

Comment:
I typically always encourage developers to not include one line `if` statements like this because it's so prone to error, and also makes it more work to add debug statements or add on to that condition, so I would personally encourage changing these to a non-one line `if`.

Reply:
Right, I'm surprised prettier didn't auto correct this... perhaps we need to enable that.

Reply:
Yeah I was thinking about opening a PR to update our linting configs.